source/eprosima-suite/eprosima_suite.rst

```diff
@@ -0,0 +1,191 @@
+:orphan:
+
+.. _eprosima_suite:
+
+eProsima Suite
+==============
+
+eProsima provides the eProsima Suite Docker image for those who want a Docker image with a set of eProsima's tools
+and libraries running on an Ubuntu platform. It can be downloaded from
+`eProsima's downloads page <https://eprosima.com/index.php/downloads-all>`_.
+
+This Docker image was built for Ubuntu 20.04 (Focal Fossa).
+
+To run this container you need Docker installed. From a terminal run
+
+.. code-block:: bash
+
+    $ sudo apt install docker.io
+
+This Docker image contains the complete eProsima Suite. This includes:
+
+- :ref:`eProsima Fast DDS libraries and examples <fast_dds_suite_examples>`: Fast DDS libraries bundled with several
```

Comment:
I guess this depends on whether you consider RTPS/DDS and CDR libraries as separate entities or as part of a single library. I was going for the former when i wrote these lines, but i can see why it might be the other way around.